src/components/Switch/Switch.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+import Switch from './Switch';
+
+describe('<Switch />', () => {

```

Comment:
Like the checkbox, it would be good to have tests for the different switch states. But this can wait until next tri. :)